webview/src/plots/components/PlotsContainer.tsx

```diff
@@ -63,22 +69,15 @@ export const PlotsContainer: React.FC<PlotsContainerProps> = ({
   currentSize,
   menu
 }) => {
-  const [size, setSize] = useState<PlotSize>(currentSize)
+  const [size, setSize] = useState<number>(currentSize || 400)
```

Comment:
Maybe I'm wrong, but isn't it recommended that when it comes to updating state based off of props, you should update with a `useEffect` hook? Like this:

```
const [size, setSize] = useState<number>(400)

  useEffect(() => {
    setCurrentSize(currentSize)
  }, [currentSize]);
```

Reply:
We're already setting the size inside the in the `changeSize` function, we don't need to wait for the size to come back from the extension. We only care about the initial one.